Change import GPG key description (#1744)
uyuni-project/uyuni-docs: modules/administration/pages/repo-metadata.adoc

.Procedure: Importing GPG keys on Clients

> . The deployment for GPG keys to the clients is done via salt states. Just apply the highstate via the {productname} {webui}.

Reviewer: ```suggestion
. Deploying GPG keys to the clients works with salt states.
Apply the highstate with the {productname} {webui}.
```
